### PR TITLE
Adding backup engine info

### PR DESCRIPTION
**Pull request by linxGnu**

### CODE REVIEW

File: tests/test_backup.rs [hunk 1]
```diff
@@ -36,6 +36,14 @@ fn backup_restore() {
             let mut backup_engine = BackupEngine::open(&backup_opts, &backup_path).unwrap();
             assert!(backup_engine.create_new_backup(&db).is_ok());
 
+            // check backup info
+            let info = backup_engine.get_backup_info();
+            assert!(info.len() > 0);
```
Comment (aleksuss) [inline tests/test_backup.rs:41]:
```suggestion
            assert!(!info.is_empty());
```

File: src/backup.rs [hunk 2]
```diff
@@ -15,10 +15,17 @@
 
 use crate::{ffi, Error, DB};
 
-use libc::c_int;
+use libc::{c_int, c_uchar};
 use std::ffi::CString;
 use std::path::Path;
 
+pub struct BackupEngineInfo {
```
Comment (aleksuss) [inline src/backup.rs:22]: Add a short description for the public struct please.

File: src/backup.rs [hunk 3]
```diff
@@ -15,10 +15,17 @@
...
+    pub timestamp: i64,
```
Comment (aleksuss) [inline src/backup.rs:23]: It'd be good to add a description to public members of a struct as well 😄

**Comment by linxGnu** [comment]
Hello @aleksuss , I addressed your comments. PTAL. Thank you

**Pull request merged**